fabric-resource-loader-v1/build.gradle

```diff
@@ -12,3 +12,47 @@ testDependencies(project, [
 	':fabric-gametest-api-v1',
 	':fabric-resource-loader-v1'
 ])
+
+// Setup 3 test mods used for testing resource sorting
+sourceSets {
+	testmodA
+	testmodB
+	testmodC
+}
+
+[sourceSets.testmodA, sourceSets.testmodB, sourceSets.testmodC].each { sourceSet ->
+	dependencies {
+		testmodImplementation sourceSet.output
+	}
+	rootProject.dependencies {
+		testmodImplementation sourceSet.output
+	}
+
+	tasks.register("${sourceSet.name}Jar", Jar) {
+		from sourceSet.output
+		archiveBaseName.set(sourceSet.name)
+	}
+}
+
+rootProject.allprojects.each { p ->
+	if (p.extensions.findByName("loom") == null) {
+		return // Skip over the meta projects
+	}
+
+	p.loom.mods.register("fabric-resource-loader-v1-testmod-a") {
```

Comment:
Why are you registering these to all projects?

Reply:
It's there in the root one since the root project has all test mods, but I don't think modules depend on other modules' test mods otherwise 🤔

Reply:
It is an `allprojects` in an `allprojects` block so, yeah, idk.